section3/reflection.md

```diff
@@ -1,17 +1,26 @@
 ## Section 3 Reflection
 
 1. What are two points from the Growth Mindset article and/or video that either resonated with you, or were brand new to you?
+  1. I hadn't heard of SMART goals before. Setting specific, measurable, achievable, relevant, & time-bound goals would make my own goals more realistic & achievable.
```

Comment:
Did you have a file for your work with [Ruby100](http://tutorials.jumpstartlab.com/projects/ruby_in_100_minutes.html#8.-hashes)